src/lib/select/select.ts

```diff
@@ -758,6 +782,45 @@ export class MatSelect extends _MatSelectMixinBase implements AfterContentInit,
     return !this._selectionModel || this._selectionModel.isEmpty();
   }
 
+  /** Show the backdrop of the overlay. */
+  private _showBackdrop() {
+    if (this.overlayDir.overlayRef.backdropElement) {
+      this.overlayDir.overlayRef.backdropElement.style.display = 'block';
+    }
+  }
+
+  /** Hide the backdrop of the overlay. */
+  private _hideBackdrop() {
+    if (this.overlayDir.overlayRef.backdropElement) {
+      this.overlayDir.overlayRef.backdropElement.style.display = 'none';
+    }
+  }
+
+  /** Initialize the overlay panel, and make it invisible. */
+  private _initializePanel() {
+    if (this.disabled || !this.options || !this.options.length) {
+      return;
+    }
+
+    this._triggerRect = this.trigger.nativeElement.getBoundingClientRect();
+    // Note: The computed font-size will be a string pixel value (e.g. "16px").
+    // `parseInt` ignores the trailing 'px' and converts this to a number.
+    this._triggerFontSize = parseInt(getComputedStyle(this.trigger.nativeElement)['font-size']);
+
+    this._calculateOverlayPosition();
+    this._changeDetectorRef.markForCheck();
+
+    // Set the font size on the panel element once it exists.
+    this._ngZone.onStable.asObservable().pipe(take(1)).subscribe(() => {
```

Comment:
Rather than waiting for `onStable` and then having a null check for the `overlayRef`, we should be able to use the `attach` on the overlay directive to know when it was attached.